src/desktop/DesktopCryptoFacade.js

```diff
@@ -24,7 +24,7 @@ export class DesktopCryptoFacade {
 		this._addEntropy()
 	}
 
-	aesEncryptObject(encryptionKey: Aes256Key, object: {}): string {
+	aesEncryptObject(encryptionKey: Aes256Key, object: number | string | boolean | $ReadOnlyArray<*> | {}): string {
```

Comment:
isn't `{}` just `any` anyway?

Reply:
No it's not its object without requirements